quay-enterprise/quay-ssl.md

```diff
@@ -103,11 +98,13 @@ $ docker restart cbe7b0fa39d8
 
 ### Test the secure connection
 
-Confirm the configuration by visiting the URL from a browser: `https://reg.example.com/`
+Confirm the configuration by visiting the URL from a browser `https://reg.example.com/`
 
 <img src="img/https-browser.png" class="img-center" alt="Browser"/>
 
-"Your Connection is not secure" means the CA is not officially and publicly trusted, but confirms that SSL is functioning properly. Check Google for how to configure your operating system and browser to trust a certificate signed by your own CA.
+
+"Your Connection is not secure" means the CA is untrusted but confirms that SSL is functioning properly. Google how to trust a CA based on the browser/operating system.
```

Comment:
Last sentence: Replace with:
"Check Google for how to configure your operating system and web browser to trust your new CA."